apps/libs/filters/src/GlobalValidationPipe.ts

```diff
@@ -5,6 +5,7 @@ export class GlobalValidationPipe extends ValidationPipe {
   constructor () {
     super({
       transform: true,
+      // forbidUnknownValues: true,
```

Comment:
reply for https://github.com/DeFiCh/jellyfish/pull/948#issuecomment-1005532103

just disabled as it's causing pg test failed
https://github.com/DeFiCh/jellyfish/blob/c271cc56402e37611e51586a254576ac216f72e8/apps/playground-api/__tests__/controllers/RpcController.test.ts#L14